paradise_papers_search/fetch_api/models/Entity.py

```diff
@@ -19,7 +19,61 @@ class Entity(DjangoNode):
     status                   = StringProperty()
     officers                 = RelationshipFrom('.Officer.Officer', 'OFFICER_OF')
     intermediaries           = RelationshipFrom('.Intermediary.Intermediary', 'INTERMEDIARY_OF')
-    addressess               = RelationshipTo('.Address.Address', 'REGISTERED_ADDRESS')
+    addresses                = RelationshipTo('.Address.Address', 'REGISTERED_ADDRESS')
     others                   = RelationshipFrom('.Other.Other', 'CONNECTED_TO')
 
+    @property
+    def serialize(self):
+        return {
+            'properties': {
+                'sourceID': self.sourceID,
+                'address': self.address,
+                'jurisdiction': self.jurisdiction,
+                'service_provider': self.service_provider,
+                'countries': self.countries,
+                'jurisdiction_description': self.jurisdiction_description,
+                'valid_until': self.valid_until,
+                'ibcRUC': self.ibcRUC,
+                'name': self.name,
+                'country_codes': self.country_codes,
+                'incorporation_date': self.incorporation_date,
+                'node_id': self.node_id,
+                'status': self.status,
+            },
+        }
+
+    @property
+    def serialize_connections(self):
+        return [
+            {
+                'nodes_type': 'officer',
+                'nodes_related': [officer.serialize for officer in self.officers.all()],
```

Comment:
Excellence work mixing Python list comprehension and Neomodel Object Graph Mapper. 🍰